### PR TITLE
Installing IMOS Compliance Checker Plugin

### PR DESCRIPTION
**Pull request by bhasin85**

### CODE REVIEW

File: cookbooks/imos_po/attributes/netcdf_checker.rb [hunk 1]
```diff
@@ -2,3 +2,6 @@
 default['imos_po']['netcdf_checker']['repo']       = "https://github.com/ioos/compliance-checker.git"
 default['imos_po']['netcdf_checker']['branch']     = "master"
 default['imos_po']['netcdf_checker']['executable'] = "/usr/local/bin/netcdf-checker"
+
+# IMOS Compliance Checker Plugin
+default['imos_po']['cc_plugin_imos']['dir'] = "/mnt/ebs/data-services/lib/cc_plugin_imos"
```
Comment (danfruehauf) [inline cookbooks/imos_po/attributes/netcdf_checker.rb]: Newline + `::File.join(default['imos_po']['data_services']['dir'], "...")`

And if all attributes are called `netcdf_checker`, then introducing a `cc_plugin_imos` attribute makes no sense.

**Comment by lbesnard** [comment]
@lwgordonimos I don't know why JQ was not installed by @danfruehauf but I bet there was a reason, I recall having a brief chat about this, so maybe watch out for potential issues. I always had to install it manually on my pobox

**Comment by lwgordonimos** [comment]
@lbesnard It's a single standalone binary, so can't cause any issues to any existing stuff. The reason is most likely that it is more appropriate to use a native library if running out of code, so will confirm the purpose of it with @bhasin85

**Comment by lwgordonimos** [comment]
@lbesnard Ugh, sorry realised that commit was in the wrong branch and has nothing to do with this PR

**Comment by danfruehauf** [comment]
@lbesnard No reason, laziness. It did not exist in stock repositories last I checked. Should be installed by `imos_po`, not `imos_jenkins`. And clean this PR, seriously.

File: cookbooks/imos_po/recipes/netcdf_checker.rb [hunk 2]
```diff
@@ -19,3 +19,12 @@
 link node['imos_po']['netcdf_checker']['executable'] do
   to ::File.join(node['imos_po']['netcdf_checker']['dir'], "cchecker.py")
 end
+
+Chef::Log.info("Installing IMOS Compliance Checker Plugin")
+execute "install_cc_plugin_imos" do
+  cwd node['imos_po']['netcdf_checker']['cc_plugin_dir']
+  command "pip install -e ."
```
Comment (anguss00) [inline cookbooks/imos_po/recipes/netcdf_checker.rb]: is it safer to specify requirements.txt here?

**Comment by anguss00** [comment]
Just one comment here @bhasin85. Also, it would be great to squash some of the commits here.

**Comment by bhasin85** [comment]
@anguss00  I was thinking if we can use the cool [github feature](https://help.github.com/articles/about-pull-request-merge-squashing/) below. Does that sounds good to you?

squash-and-merge

**Comment by bhasin85** [comment]
Thanks @anguss00  :ok_hand:

**Pull request merged**